app/models/invitation.rb

```diff
@@ -0,0 +1,35 @@
+class Invitation
+  include ActiveModel::Model
+  include ActiveModel::Validations
+
+  require "http"
+
+  attr_accessor :email
+  validates_presence_of :email
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.